[2018-04-18 06:00:44,347] {models.py:167} INFO - Filling up the DagBag from /home/ubuntu/workspace/airflow/dags/Main_dag.py
[2018-04-18 06:00:45,232] {base_task_runner.py:112} INFO - Running: ['bash', '-c', u'airflow run Jobsv31.doubleclick_search_keyword create_folder_with_current_date 2018-04-17T01:00:00 --job_id 6082 --raw -sd DAGS_FOLDER/Main_dag.py']
[2018-04-18 06:00:46,153] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:46,153] {__init__.py:57} INFO - Using executor LocalExecutor
[2018-04-18 06:00:46,318] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:46,318] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/Grammar.txt
[2018-04-18 06:00:46,362] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:46,361] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/PatternGrammar.txt
[2018-04-18 06:00:46,714] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:46,714] {models.py:167} INFO - Filling up the DagBag from /home/ubuntu/workspace/airflow/dags/Main_dag.py
[2018-04-18 06:00:47,475] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,474] {models.py:1126} INFO - Dependencies all met for <TaskInstance: Jobsv31.doubleclick_search_keyword.create_folder_with_current_date 2018-04-17 01:00:00 [queued]>
[2018-04-18 06:00:47,483] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,482] {models.py:1126} INFO - Dependencies all met for <TaskInstance: Jobsv31.doubleclick_search_keyword.create_folder_with_current_date 2018-04-17 01:00:00 [queued]>
[2018-04-18 06:00:47,483] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,482] {models.py:1318} INFO - 
[2018-04-18 06:00:47,483] {base_task_runner.py:95} INFO - Subtask: --------------------------------------------------------------------------------
[2018-04-18 06:00:47,483] {base_task_runner.py:95} INFO - Subtask: Starting attempt 1 of 4
[2018-04-18 06:00:47,484] {base_task_runner.py:95} INFO - Subtask: --------------------------------------------------------------------------------
[2018-04-18 06:00:47,484] {base_task_runner.py:95} INFO - Subtask: 
[2018-04-18 06:00:47,508] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,508] {models.py:1342} INFO - Executing <Task(BashOperator): create_folder_with_current_date> on 2018-04-17 01:00:00
[2018-04-18 06:00:47,562] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,562] {bash_operator.py:71} INFO - tmp dir root location: 
[2018-04-18 06:00:47,562] {base_task_runner.py:95} INFO - Subtask: /tmp
[2018-04-18 06:00:47,564] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,563] {bash_operator.py:80} INFO - Temporary script location :/tmp/airflowtmpUYv2eU//tmp/airflowtmpUYv2eU/create_folder_with_current_datenFeLZY
[2018-04-18 06:00:47,564] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,564] {bash_operator.py:81} INFO - Running command: /home/ubuntu/workspace/airflow/bash/create_folder_with_current_date.sh doubleclick_search 20180417
[2018-04-18 06:00:47,572] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,572] {bash_operator.py:90} INFO - Output:
[2018-04-18 06:00:47,579] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,579] {bash_operator.py:94} INFO - ================test!!!!!!!!==========
[2018-04-18 06:00:47,584] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,584] {bash_operator.py:94} INFO - Pseudo-terminal will not be allocated because stdin is not a terminal.
[2018-04-18 06:00:47,845] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,844] {bash_operator.py:94} INFO - Welcome to Ubuntu 16.04.3 LTS (GNU/Linux 4.4.0-1052-aws x86_64)
[2018-04-18 06:00:47,845] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,844] {bash_operator.py:94} INFO - 
[2018-04-18 06:00:47,845] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,844] {bash_operator.py:94} INFO - * Documentation:  https://help.ubuntu.com
[2018-04-18 06:00:47,845] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,844] {bash_operator.py:94} INFO - * Management:     https://landscape.canonical.com
[2018-04-18 06:00:47,846] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,844] {bash_operator.py:94} INFO - * Support:        https://ubuntu.com/advantage
[2018-04-18 06:00:47,846] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,844] {bash_operator.py:94} INFO - 
[2018-04-18 06:00:47,846] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,844] {bash_operator.py:94} INFO - Get cloud support with Ubuntu Advantage Cloud Guest:
[2018-04-18 06:00:47,846] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,844] {bash_operator.py:94} INFO - http://www.ubuntu.com/business/services/cloud
[2018-04-18 06:00:47,846] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,844] {bash_operator.py:94} INFO - 
[2018-04-18 06:00:47,846] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,844] {bash_operator.py:94} INFO - 104 packages can be updated.
[2018-04-18 06:00:47,846] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,845] {bash_operator.py:94} INFO - 0 updates are security updates.
[2018-04-18 06:00:47,846] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,845] {bash_operator.py:94} INFO - 
[2018-04-18 06:00:47,846] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,845] {bash_operator.py:94} INFO - 
[2018-04-18 06:00:47,846] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,845] {bash_operator.py:94} INFO - *** System restart required ***
[2018-04-18 06:00:47,853] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,853] {bash_operator.py:94} INFO - ========executing this file in ETL!!! ===============
[2018-04-18 06:00:47,853] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,853] {bash_operator.py:94} INFO - /home/ubuntu/csv_files/doubleclick_search/20180417
[2018-04-18 06:00:47,858] {base_task_runner.py:95} INFO - Subtask: [2018-04-18 06:00:47,857] {bash_operator.py:97} INFO - Command exited with return code 0
[2018-04-18 06:00:50,245] {jobs.py:2083} INFO - Task exited with return code 0
